.github/workflows/e2e-tests.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
     needs: build
     strategy:
       matrix:
-        cluster-type: ["kops", "eksctl"]
+        cluster-type: ["eksctl", "kops"]
```

Comment:
Why the swap here?

Reply:
unintentional, I had removed it in a previous commit to test kops only and then added it back in but in reverse order